index.js

```diff
@@ -146,10 +148,30 @@ Liftoff.prototype.buildEnvironment = function (opts) {
     configPath: configPath,
     configBase: configBase,
     modulePath: modulePath,
-    modulePackage: modulePackage || {}
+    modulePackage: modulePackage || {},
+    configFiles: configFiles
   };
 };
 
+// use rechoir to autoload any required modules
+function registerLoader(extensions, configPath, cwd, eventEmitter) {
+  if (configPath) {
```

Comment:
instead of this nested `if` statement, check `if (!configPath)` and return early